F/FastANI/build_tarballs.jl

```diff
@@ -0,0 +1,33 @@
+using BinaryBuilder, Pkg
+
+name = "FastANI"
+version = v"1.33.0"
+sources = [
+    GitSource("https://github.com/ParBLiSS/FastANI.git", "6531da0957344d2a56e3eeb9bb83fea02cc4abe0"),
+]
+
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/FastANI
+mkdir -p "${bindir}"
+c++ -O3 -DNDEBUG -std=c++11 -Isrc -fopenmp   src/cgi/core_genome_identity.cpp -o "${bindir}/fastANI${exeext}" -lgsl -lgslcblas -lz -lm  
+"""
+
+
+platforms = supported_platforms()
+platforms = filter(!Sys.iswindows, platforms)
```

Comment:
Unless they do something non-portable in the single file you're compiling, this may even work on Windows without any extra effort.  Worth a try.
```suggestion
```

Reply:
Nevermind, they did manage to use non-portable functions within 100 lines of code.